source/wp-content/plugins/wporg-internal-notes/src/store/reducer.js

```diff
@@ -44,14 +54,28 @@ export const reducer = ( state = DEFAULT_STATE, { note, notes, noteId, totalNote
 				totalNotes,
 				notes,
 			};
+		case PREPEND_NOTES:
+			const prependedNoteIds = notes.map( ( prependedNote ) => prependedNote.id );
+
+			return {
+				...state,
+				totalNotes: state.totalNotes + notes.length,
```

Comment:
Is there a reason to not use `totalNotes` from the action?

Reply:
Nope. `totalNotes` totally works there. b07f05a